ynr/apps/people/management/commands/remove_inactive_person_links.py

```diff
@@ -0,0 +1,49 @@
+import requests
+from django.core.management.base import BaseCommand
+from people.models import Person
+
+
+class Command(BaseCommand):
+    """
+    Test and remove inactive or dead links from Person objects.
+    """
+
+    def handle(self, *args, **options):
+        """
+        Iterate over all Person objects and check if the
+        person identifier urls return a 200 status code.
+        """
+        inactive_links = []
+        people = Person.objects.all()
+        for person in people:
+            person_identifiers = person.get_all_identifiers
+            person_identifiers = [
+                identifier
+                for identifier in person_identifiers
+                if identifier.value.startswith("http")
+            ]
+
+            if person_identifiers:
+                for identifier in person_identifiers:
+                    resp = None
+                    try:
+                        resp = requests.head(identifier.value).status_code
+                    except requests.exceptions.RequestException as e:
+                        self.stdout.write(
+                            f"Request exception: {e} for {person.name}"
+                        )
+                        pass
+                    if resp in [404, 500, 502, 503, 504]:
+                        self.stdout.write(
+                            f"Status code: {resp} for {person.name} {identifier.value}"
+                        )
+                        inactive_links.append(identifier)
+                        for i, ident in enumerate(person_identifiers):
+                            if ident == identifier:
+                                person_identifiers.pop(i)
+                                print(
+                                    "Inactive links found for {person.name} and deleted"
```

Comment:
This doesn't actually delete? Two options:

1. Just make a report of links to delete and ask humans to do it. This is slow, but allows for double checking. In this case, can you output as TSV so we can copy into a Google Sheet?
2. Delete them in code, and in that case you'll need to use CandidateBot to make the version history correct.